include/links-directory.php

```diff
@@ -44,14 +40,13 @@ public function __construct( &$model ) {
 	 * @return void
 	 */
 	public function init() {
```

Comment:
I believe that we can simplify and suppress this method to avoid one function call. `add_permastruct()` could be called directly from the constructor.